.github/workflows/dd-build.yaml

```diff
@@ -0,0 +1,143 @@
+name: Build and Push etcd releases
+
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    tags:
+    # Push events on datadog tags
+    - "*-dd*"
+permissions: write-all
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: ["linux/arm64","linux/amd64"]
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Set release version environment variable
+      run: echo RELEASE_VERSION=${GITHUB_REF#refs/tags/} >> $GITHUB_ENV
+      env:
+        GITHUB_REF: ${{ github.ref }}
+    - name: Set target platform environment variable
+      run: echo SANITIZED_TARGET_PLATFORM=${KUBE_BUILD_PLATFORM/\//-} >> $GITHUB_ENV
+      env:
+        KUBE_BUILD_PLATFORM: ${{ matrix.platform }}
```

Comment:
This will probably not work in that case
I would expect a GOOS or something to that effect.

`KUBE_BUILD_PLATFORM ` is the var that is used by the kube builds script to build the right thing, but for ETCD it's probably not what you're looking for.

Reply:
Removed.